tools/debian/control

```diff
@@ -19,7 +19,8 @@ Build-Depends: debhelper-compat (= 13),
                libpam0g-dev,
                libpcp-import1-dev,
                libpcp-pmda3-dev,
-               systemd,
+               pkgconf,
+               systemd-dev | systemd,
```

Comment:
```suggestion
               systemd-dev | systemd (<< 254),
```

To avoid being faced with a `systemd` package that does not depend on `systemd-dev` and therefore lacks `systemd.pc` later.

Reply:
Good idea, yes -- committed locally with `<< 253-2~`, I'll push it once our ubuntu-2204 image is sorted out.